project/AppDependencies.scala

```diff
@@ -4,19 +4,19 @@ import sbt._
 object AppDependencies {
 
   val compile = Seq(
-    "uk.gov.hmrc" %% "bootstrap-frontend-play-28" % "7.19.0",
-    "uk.gov.hmrc" %% "play-frontend-hmrc" % "7.14.0-play-28",
+    "uk.gov.hmrc" %% "bootstrap-frontend-play-28" % "7.22.0",
```

Comment:
Could you please create a val like below and use instead of hardcoded 7.22.0 ? Also use bootstrapVersion at line 16?

private val bootstrapVersion = "7.22.0"

Reply:
Done